src/SFA.DAS.EmployerFinance/SFA.DAS.EmployerFinance.csproj

```diff
@@ -53,6 +53,7 @@
     <ProjectReference Include="..\SFA.DAS.Account.Api.Client\SFA.DAS.EAS.Account.Api.Client.csproj" />
     <ProjectReference Include="..\SFA.DAS.EAS.Account.Api.Types\SFA.DAS.EAS.Account.Api.Types.csproj" />
     <ProjectReference Include="..\SFA.DAS.EAS.Infrastructure.Interfaces\SFA.DAS.EAS.Infrastructure.Interfaces.csproj" />
+    <ProjectReference Include="..\SFA.DAS.EmployerApprenticeshipsService.Domain\SFA.DAS.EAS.Domain.csproj" />
```

Comment:
Has this been added in accidentally?  It'll need removing so we can pull the projects into different solutions.

Reply:
yes that's a mistake, fixed